libs/git_wrapper/Git_wrapper.ml

```diff
@@ -335,23 +340,27 @@ let status ~cwd ~commit =
     renamed = !renamed;
   }
 
-let is_git_repo cwd =
-  let cmd = (git, [ "-C"; !!cwd; "rev-parse"; "--is-inside-work-tree" ]) in
+let is_git_repo ?cwd () =
+  let cmd = (git, cd cwd @ [ "rev-parse"; "--is-inside-work-tree" ]) in
```

Comment:
This is wrong. The goal o Cmd.t is actually to make things cleaner and forbid shell execution and
be precise on the command executed.
Please revert those changes.

Reply:
`cd cwd` is not a shell command, it appends the `-C ...` option is `cwd` is specified.